__tests__/endpoint.test.js

```diff
@@ -411,4 +411,25 @@ describe('PATCH/api/articles/:article_id', () => {
         
       });
     })
+  })
+
+describe('DELETE/api/comments/:comment_id', () => {
+
+    test('sends back status code 204 when comment is deleted', () => {
+  
+      return request(app).delete("/api/comments/5").expect(204)
```

Comment:
It would be nice to have an assertion to confirm that the response is undefined here, just to be super explicit.